provisionersdk/proto/provisioner.proto

```diff
@@ -57,8 +57,16 @@ message RichParameterValue {
     string value = 2;
 }
 
+// ExpirationPolicy defines the policy for expiring unclaimed prebuilds.
+// If a prebuild remains unclaimed for longer than ttl seconds, it is deleted and
+// recreated to prevent staleness.
+message ExpirationPolicy {
+	int32 ttl = 1;
+}
+
 message Prebuild {
     int32 instances = 1;
+	ExpirationPolicy expiration_policy = 2;
```

Comment:
I'm assuming this will be joining the Provisioner API version 1.6 -- if so, please document it in `provisionerd/proto/version.go`.